Core/HLE/ReplaceTables.cpp

```diff
@@ -1043,6 +1043,41 @@ static int Hook_sdgundamggenerationportable_download_frame() {
 	return 0;
 }
 
+static int Hook_atvoffroadfurypro_download_frame() {
+	const u32 fb_address = currentMIPS->r[MIPS_REG_A1];
+	if (Memory::IsVRAMAddress(fb_address)) {
+		gpu->PerformMemoryDownload(fb_address, 0x00088000);
```

Comment:
In that case,that function is inside a loop,seems like the total size = <code>MIPS_REG_A2</code> * <code>MIPS_REG_S3</code>,and we should hook the caller instead of this one?similar to Blazin' Trails.